xclim/indices.py

```diff
@@ -378,22 +476,70 @@ def heating_degree_days(tas, freq='YS', thresh=17):
 
 @valid_daily_max_temperature
 def hot_days(tasmax, thresh=30, freq='YS'):
-    """Number of very hot days.
+    r"""Number of very hot days.
+
+    Number of days with max temperature exceeding a base threshold.
 
-    The number of days exceeding a threshold. """
+    Parameters
+    ----------
+    tasmax : xarrray.DataArray
+      Maximum daily temperature [℃] or [K]
+    thresh : float
+      Threshold temperature on which to base evaluation [℃] or [K]
+    freq : str, optional
+      Resampling frequency.
+
+    Returns
+    -------
+    xarray.DataArray
+      Number of hot days.
+    """
     hd = (tasmax > K2C + thresh) * 1
     return hd.resample(time=freq).sum(dim='time')
 
 
 @valid_daily_max_temperature
 def ice_days(tasmax, freq='YS'):
-    """Number of days where the daily maximum temperature is below 0℃."""
+    r"""Number of freezing days
+
+    Number of days where daily maximum temperature does not exceed 0℃.
```

Comment:
I feel negative clauses makes understanding more difficult (does not exceed).

Reply:
Got it.